bin/critterget.py

```diff
@@ -1032,8 +1091,97 @@ def getCrashCounts(appId, appName):
     return
 
 
+def get_exception_counts(app_id, app_name):
+    """
+    Get the number of daily handled exceptions for a given app.
+
+    :param app_id: string for API call
+    :param app_name: string for Splunk
+    :return: None
+    """
+
+    exception_data = apicall("app/exception/counts/{}".format(app_id))
+
+    print_string = u''
+
+    try:
+        for series in exception_data[DATA]:
+            print_string += u'({},{}),'.format(series[DATE], series[VALUE])
+
+        print (u'{} MessageType=ExceptionCounts appName="{}" appId="{}" DATA '
+               u'{}').format(DATETIME_OF_RUN, app_name, app_id, print_string)
+
+    except KeyError as e:
+        print (u'{} MessageType="ApteligentError" Error: Could not access '
+               u'{} in {}.').format(DATETIME_OF_RUN,
+                                    str(e),
+                                    'get_exception_counts')
+        return None
+
+
+def get_exception_details(app_id, app_name):
+    """
+    Get paginated exception summary data and pass it to Splunk
+
+    :param app_id: string
+    :param app_name: string
+    :return: None
+    """
+    exceptions = get_all_pages(u'exception/paginatedtable/{}'.format(app_id),
+                               ERRORS)
+    for exception in exceptions:
+        printstring = (u'{} MessageType="ExceptionDetail" appId={} appName={} '
+                       u'exceptionHash="{}"').format(
+                           DATETIME_OF_RUN,
+                           app_id,
+                           app_name,
+                           exception['hash']
+                       )
+        for key, value in exception.iteritems():
+            if key == 'class_name':
+                continue
+            if key == 'hash':
+                continue
+            elif value:
+                printstring += u'{}="{}"'.format(key, value)
+        print printstring
+
+
+def get_all_pages(base_url, param_to_get=None):
```

Comment:
I'd love your input on this, as far as making this function more generic and making it able to be used for crash as well.